app/code/core/Mage/Oauth2/Block/Adminhtml/Client/Edit.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+class Mage_Oauth2_Block_Adminhtml_Client_Edit extends Mage_Adminhtml_Block_Widget_Form_Container
+{
+    protected $_model;
+
+    /**
+     * Constructs the object and initializes the block group, controller, and mode.
+     * Updates the save and delete buttons with localized labels.
+     * Removes the delete button if the user is not allowed to perform the delete action.
+     * Adds a save and continue button with a localized label and onclick event.
+     * Adds a form script to submit the form with a specific action.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        $this->_blockGroup = 'oauth2';
+        $this->_controller = 'adminhtml_client';
+        $this->_mode = 'edit';
+
+        $this->_updateButton('save', 'label', Mage::helper('oauth')->__('Save'));
```

Comment:
First, I was like: don't use `Mage::helper('oauth')`, use `Mage::helper('oauth2')`, instead. (because we might want to disable the oauth module in favor of oauth2?

then i was like: doesn't blocks have their own `$this->helper` function? So wouldn't `$this->helper('oauth2')` be better?

but then i was like: why use `Mage::helper()->__()` for when you can just use `$this->__()` ?